app/views/courses/edit.html.erb

```diff
@@ -0,0 +1,8 @@
+<center>
+    <br>
+    <div>
+        <h1>Edit Course</h1>
+        <hr>
+        <%= render partial: 'form', locals: {cancel_path: user_course_path(@user, @course), cancel_text: "Cancel"} %>
+    </div>
+</center>
```

Comment:
I don't actually know why or when GitHub wants newlines at the end of files, but it says there should be one here, and in a few other files

Reply:
Github wants newlines at the end of all files because the UNIX standard says to end each file with a newline--it is a generally accepted style thing, and in very, very rare situations involving multiple, consecutively run shell scripts, it can actually change functionality.